index.html

```diff
@@ -418,6 +419,49 @@ <h2>Transferable MediaStreamTrack</h2>
       <p>The underlying source is supposed to be kept alive between the transfer and transfer-receiving steps, or as long as the data holder is alive.
       In a sense, between these steps, the data holder is attached to the underlying source as if it was a track.</p>
     </div>
+    <h3>MediaStreamTrack frame counters</h3>
```

Comment:
Add a few words about why frame counters are nice to have? (Like: Frame counters allow you to tell why the frame rate experienced is different from the expected one - for instance, the camera may be slowing down frame production because it's dark, the settings applied may produce decimation of frames, or the system may be CPU starved and therefore dropping frames.)

Reply:
Done!